WebSuggest/websuggest.py

```diff
@@ -332,19 +341,33 @@ def on_suggest(self, user_input, items_chain):
             #item.set_data_bag(user_input)
             suggestions.append(item)
 
-        if suggestions:
-            self.set_suggestions(suggestions, kp.Match.ANY, kp.Sort.NONE)
+        if not provider_suggestions:  # change default item
+            suggestions[0].set_short_desc("No suggestions found (default action: {})".format(
+                                profile['default_action']))

```

Comment:
Please indent to align with above parenthesis. `profile[...` should be aligned with `"No suggestions ...` (I know it may not be the case everywhere in the repo)